test/unit/dialects/postgres/query-generator.test.js

```diff
@@ -291,7 +291,27 @@ if (dialect.match(/^postgres/)) {
           expectation: 'SELECT * FROM "myTable" AS "myTable" ORDER BY "myTable"."id" DESC;',
           context: QueryGenerator,
           needsSequelize: true
+        },{
+          title: 'uses limit 0',
+          arguments: ['myTable', {limit: 0}],
+          expectation: 'SELECT * FROM "myTable" LIMIT 0;',
+          context: QueryGenerator
         }, {
+         title: "uses limit '0'",
+         arguments: ['myTable', {limit: '0'}],
+         expectation: "SELECT * FROM \"myTable\" LIMIT '0';",
+         context: QueryGenerator
+       },{
+         title: 'uses offset 0',
+         arguments: ['myTable', {offset: 0}],
+         expectation: 'SELECT * FROM "myTable" OFFSET 0;',
+         context: QueryGenerator
+       }, {
+        title: "uses offset '0'",
+        arguments: ['myTable', {offset: '0'}],
+        expectation: "SELECT * FROM \"myTable\" OFFSET '0';",
+        context: QueryGenerator
+      },{
```

Comment:
You can remove the string tests from both dialects, we don't official support limit/offset as strings anymore.